lib/src/base-styles/_nx-lists.scss

```diff
@@ -128,109 +136,76 @@
  * Action or Clickable Lists
  */
 .nx-list--clickable {
-  .nx-list__title {
-    color: $nx-text-color;
-  }
-
   .nx-list__item {
-    @include semi-bold();
-
-    color: $nx-text-color;
-
     &.selected {
-      background-color: $nx-grey-3;
-      color: $nx-text-color;
-      pointer-events: none;
+      &, &:hover {
+        background-color: #d3e6ff;
+      }
     }
 
-    &:not(.nx-list__item--empty):not(.nx-disabled):not([disabled]) {
-      &:hover, &:focus {
-        background-color: $nx-color-blue-96;
-        border-top: $nx-border-row-hover;
-        border-bottom: $nx-border-row-hover;
-        color: $nx-text-color;
-        cursor: pointer;
-        outline-style: none; // removes :focus outline
-
-        .fa-chevron-right {
-          color: $nx-color-blue-41;
-        }
-      }
+    &:hover {
+      background-color: #f1f3fa;
+      cursor: pointer;
+      outline-style: none; // removes :focus outline
```

Comment:
Is this still needed now that we have `outline: none` on `:focus`?

Reply:
> Is this still needed now that we have `outline: none` on `:focus`?

Nope: https://github.com/sonatype/sonatype-react-shared-components/pull/141/commits/b88e7fd49225ff1ad18a4d0c442ab7664e5e693c